### PR TITLE
add kimpy

### PR DESCRIPTION
**Pull request by jan-janssen**
Please do not merge this just now as there is an open discussion about the licensing: 
https://github.com/openkim/kimpy/issues/1

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/kimpy/build.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (marcelotrevisani) [inline recipes/kimpy/build.sh]: Could you please remove this shebang?
``conda-build`` already handles it
```suggestion
```

File: recipes/kimpy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - pybind11
+    - kim-api
+  run:
+    - python
+    - numpy
+    - pytest
```
Comment (marcelotrevisani) [inline recipes/kimpy/meta.yaml]: Do you really need ``pytest`` to run this project??
```suggestion
```

**Comment by jan-janssen** [comment]
The license was added to the upstream repository and included in this recipe. https://github.com/openkim/kimpy/commit/d3b93c3ebf56fcba80fc20f6f24bac044b9e5fe2 It is now ready to be reviewed.

**Comment by jan-janssen** [comment]
@conda-forge/staged-recipes, ready for review

**Pull request merged**